receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -42,6 +56,53 @@ func (s *brokerScraper) shutdown(context.Context) error {
 	return nil
 }
 
+func (s *brokerScraper) scrapeMetric(now pcommon.Timestamp, allMetrics saramaMetrics, brokerID int64, prefix string) {
+	key := fmt.Sprint(prefix, brokerID)
+
+	if metric, ok := allMetrics[key]; ok {
+		switch prefix {
+		case "consumer-fetch-rate-for-broker-":
+			if v, ok := metric["mean.rate"].(float64); ok {
```

Comment:
Is the `mean.rate` the only option provided by the sarama library? If we can get a cumulative counter, that would be much better. The metric can be easily aggregated and converted to rate/s with accurate values.